src/manual/design/devel/best-practices/testing.md

```diff
@@ -46,7 +46,10 @@ Team members and dedicated contributors will want to know how to set up a full s
 Include a section in the README describing how to find or generate credentials for each service.
 
 For Taskcluster credentials, include the necessary credentials in a role named `project:taskcluster:tests:<projectName>` (e.g., `project:taskcluster:tests:taskcluster-queue`).
-Then instruct users to run `eval $(taskcluster-cli signin --scopes assume:project:taskcluster:tests:$PROJECT)` to get the scopes to run the tests, noting that they must themselves have the given role or this won't work.
+Then instruct users how to get those scopes:
+
+> To run *all* tests, you will need appropriate Taskcluster credentials.
+> Using [taskcluster-cli](https://github.com/taskcluster/taskcluster-cli), run `eval $(taskcluster signi --scopes assume:project:taskcluster:tests:taskcluster-secrets)`, then run `yarn test` again.
```

Comment:
signi

Reply:
wtf, that was one of the typos I thought I fixed..